sdk/core/azure-core/test/ut/transport_adapter_base.cpp

```diff
@@ -448,6 +446,9 @@ namespace Azure { namespace Core { namespace Test {
         OPEN_EXISTING,
         FILE_FLAG_SEQUENTIAL_SCAN,
         NULL);
+#else
+    testDataPath.append("/fileData");
+    int f = open(testDataPath.data(), O_RDONLY);
```

Comment:
Need to check the open for failure.

Reply:
This is not a goal of this PR, I am just moving things around, let's take care of this in a separate PR. @vhvb1989 will you look into this, should be a simple fix, or should I create a work item?

Reply:
Although its not a goal of the PR you are touching the line of code that is broken.  Its a best practice that if you touch a line of code you should do your best to ensure that line of code is correct.  I made a bug for it

Reply:
This is tests code.
A file body stream will never open a file, it expects the file to be open already by the time the FileBodyStream is created.

Do we need to worry about this in the tests that are only checking the FileBodyStream behavior and not the file open() functions?